src/app/wallet/wallet-tx-history-item/wallet-tx-history-item.component.html

```diff
@@ -1,8 +1,8 @@
 <ng-container *ngIf="(txView$ | async) as tx">
-  <div class="flex gap-2">
+  <div class="flex items-center gap-2">
     <div>
       <div *ngIf="tx.isOutgoing"
-           class="rounded-full w-10 h-10 bg-indigo-600 flex justify-center items-center text-white">
+           class="rounded-full w-10 h-10 bg-pink-600 flex justify-center items-center text-white">
```

Comment:
Outgoing transactions should have pink icon per design, not blue. Incoming transactions already have correctly colored icon.
Before:
![Screenshot 2021-09-22 at 17 38 17](https://user-images.githubusercontent.com/12549625/134381245-50b05ac4-a882-48ae-8911-f10052707d29.png)
After:
![Screenshot 2021-09-22 at 17 40 26](https://user-images.githubusercontent.com/12549625/134381171-ddf5036e-dfb4-425b-be22-29968f360a46.png)

Reply:
I've set it to indigo because it better suits our color palette.